setup.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import ast
+import re
+
+from setuptools import setup
+
+_version_re = re.compile(r'__version__\s+=\s+(.*)')
+
+with open('teamsupport/__init__.py', 'rb') as f:
+    version = str(ast.literal_eval(_version_re.search(
+        f.read().decode('utf-8')).group(1)))
+
+with open('README.rst') as readme_file:
+    readme = readme_file.read()
+
+with open('CHANGELOG.rst') as changelog_file:
+    changelog = changelog_file.read()
+
+requirements = [
+    'demands',
+    'lxml',
+    'property-caching',
+    'querylist',
+]
+
+test_requirements = [
+    'mock',
```

Comment:
Yeah, we do. Though that's more because of an internal issue where we can't use mock 1.3.0, but I guess other people might encounter the same issue when using 1.2.0.